importing/import_objects.py

```diff
@@ -309,17 +309,24 @@ def add_object(line, counter, position_decrement_due_to_rule, position_decrement
                     line[fields.index(field_key)] = duplicates_dict[field_value]
                 return add_object(line, counter, position_decrement_due_to_rule, position_decrement_due_to_section, fields,
                                   api_type, generic_type, layer, layers_to_attach,
-                                  changed_layer_names, api_call, num_objects, client)
+                                  changed_layer_names, api_call, num_objects, client, args)
             else:
                 debug_log("Not unique name problem \"%s\" - cannot change payload to use UID instead of name." % field_value, True, True)
         elif "will place the exception in an Exception-Group" in api_reply.error_message:
             return add_object(line, counter, position_decrement_due_to_rule - 1, position_decrement_due_to_section,
                               fields, api_type, generic_type, layer, layers_to_attach,
-                              changed_layer_names, api_call, num_objects, client)
+                              changed_layer_names, api_call, num_objects, client, args)
 
         position_decrement_due_to_rule += 1
 
         debug_log(log_err_msg, True, True)
+        if args.strict:

```

Comment:
what if args is None? better to add check for this

Reply:
Done